src/styles/base/_dropdown.scss

```diff
@@ -0,0 +1,61 @@
+.Dropdown-root {
```

Comment:
In this case, i am using 'react-dropdown' component which styles are implicit. If i change the name to lowercase, the component would not recognize it.